nixos/modules/installer/cd-dvd/sd-image.nix

```diff
@@ -1,6 +1,7 @@
 { config, ... }:
 {
   imports = [
+    ../../profiles/all-hardware.nix
```

Comment:
Why is this added to this "alias-like" file?

Reply:
No good reason, it just felt bad to not limit the impact of this change.